app/views/_includes/applications/english-language-qualification.njk

```diff
@@ -1,6 +1,6 @@
 {% set item = application.englishLanguageQualification %}
-<div class="govuk-!-margin-bottom-8">
-  <h2 class="govuk-heading-m">English as a foreign language</h2>
+<h2 class="govuk-heading-m govuk-!-margin-top-5">English as a foreign language</h2>
```

Comment:
We could make future iterations easier if we specify a heading class rather than an amalgamation of overrides